py/dml/compat.py

```diff
@@ -198,6 +198,30 @@ class suppress_WLOGMIXUP(CompatFeature):
     short = "Suppress the warning 'WLOGMIXUP' by default"
     last_api_version = api_6
 
+@feature
+class suppress_WBIGUNROLL(CompatFeature):
+    '''This compatibility feature makes it so the warning `WBIGUNROLL` is
+    suppressed by default. `WBIGUNROLL` warns about `#foreach` and `#select`
+    statements that compile to large unrolled loops &mdash; for more
+    information, see the documentation of `WBIGUNROLL` in the
+    [Messages](messages.html) section.
```

Comment:
better with a direct link, add `<a id=` on the destination if needed

Reply:
Oh, seems that `messages.html#dt:wbigunroll` would work out-of-the-box

Reply:
whaaat
That's a thing? Wow.

Reply:
Though `validate_md_links.py` hates it. I guess we could patch it to tolerate `#dt:blabla` if any line in the target file starts with `* blabla`?

Reply:
> Oh, seems that messages.html#dt:wbigunroll would work out-of-the-box
wait. What experiment did you do, that made you say this?
If we look at what `messages_to_md.py` does...
```
        f.write(f"  <dt><b>\n\n{fmt_message(m)} [{m.__name__}]</b></dt>\n")
```
Looking at this, I don't understand why the hypelink syntax would pick out and work specifically with `m__name__`.

But! Maybe the better approach is to just tweak `messages_to_md.py` to generate those anchors, so we don't have to hand-write them.

Reply:
> What experiment did you do, that made you say this?
I looked at the generated messages.html, and also typed `file:///path/to/messages.html#dt:wsystemc` into my web browser

Reply:
... huh!